src/fides/api/task/deprecated_graph_task.py

```diff
@@ -0,0 +1,314 @@
+        format_data_use_map_for_caching(
+            {
+                coll_address: gt.execution_node.connection_key
+                for (coll_address, gt) in env.items()
+            },
+            connection_configs,
+        )
+    )
+
+    v = delayed(get(dsk, TERMINATOR_ADDRESS, num_workers=1))
+    return v.compute()
+
+
+def update_erasure_mapping_from_cache(
+    dsk: Dict[CollectionAddress, Union[Tuple[Any, ...], int]], resources: TaskResources
+) -> None:
+    """On pause or restart from failure, update the dsk graph to skip running erasures on collections
+    we've already visited. Instead, just return the previous count of rows affected.
+
+    If there's no cached data, the dsk dictionary won't change.
+    """
+    cached_erasures: Dict[str, int] = resources.get_all_cached_erasures()
+
+    for collection_name in cached_erasures:
+        dsk[CollectionAddress.from_string(collection_name)] = cached_erasures[
+            collection_name
+        ]
+
+
+def run_erasure_request_deprecated(  # pylint: disable = too-many-arguments
+    privacy_request: PrivacyRequest,
+    policy: Policy,
+    graph: DatasetGraph,
+    connection_configs: List[ConnectionConfig],
+    identity: Dict[str, Any],
+    access_request_data: Dict[str, List[Row]],
+    session: Session,
+) -> Dict[str, int]:
+    """Deprecated: Run an erasure request sequentially in-memory using Dask"""
+    traversal: Traversal = Traversal(graph, identity)
+    resources = TaskResources(
+        privacy_request, policy, connection_configs, EMPTY_REQUEST_TASK, session
+    )
+
+    def collect_tasks_fn(
+        tn: TraversalNode, data: Dict[CollectionAddress, GraphTask]
+    ) -> None:
+        """Run the traversal, as an action creating a GraphTask for each traversal_node."""
+        if not tn.is_root_node():
+            # Mock a RequestTask object in memory
+            resources.privacy_request_task = tn.to_mock_request_task()
+            data[tn.address] = GraphTask(resources)
+
+    env: Dict[CollectionAddress, GraphTask] = {}
+    # Modifies env in place
+    traversal.traverse(env, collect_tasks_fn)
+    erasure_end_nodes = list(graph.nodes.keys())
+
+    def termination_fn(*dependent_values: int) -> Dict[str, int]:
+        """
+        The erasure order can be affected in a way that not every node is directly linked
+        to the termination node. This means that we can't just aggregate the inputs directly,
+        we must read the erasure results from the cache.
+        """
+        return resources.get_all_cached_erasures()
+
+    access_request_data[ROOT_COLLECTION_ADDRESS.value] = [identity]
+
+    dsk: Dict[CollectionAddress, Any] = {
+        k: (
+            t.erasure_request,
+            access_request_data.get(
+                str(k), []
+            ),  # Pass in the results of the access request for this collection
+            *_evaluate_erasure_dependencies(t, erasure_end_nodes),
+        )
+        for k, t in env.items()
+    }
```

Comment:
https://github.com/ethyca/fidesops/pull/1168

Previous dsk graph also passed in original input data we used for the access graph, which was helpful for cases like the EmailConnector where the access request doesn't actually retrieve data.  However, this version of the email connector was later replaced and this data was not in use. 

Removing this also prevents us from having to retain this concept in DSR 3.0

```
   dsk: Dict[CollectionAddress, Any] = {
            k: (
                t.erasure_request,
                access_request_data[
                    str(k)
                ],  # Pass in the results of the access request for this collection
                *[
                    access_request_data[
                        str(upstream_key)
                    ]  # Additionally pass in the original input data we used for the access request. It's helpful in
                    # cases like the EmailConnector where the access request doesn't actually retrieve data.
                    for upstream_key in t.input_keys
                ],
            )
            for k, t in env.items()
        }
```